src/back-end/lib/routers/auth.ts

```diff
@@ -402,17 +437,18 @@ async function establishSessionWithClaims(
       return null;
   }
 
-  let username = getString(claims, "preferred_username");
-  const idpId = getString(claims, "idp_id");
-
-  // Strip the vendor/gov suffix if present.  We want to match and store the username without suffix.
-  if (
-    (username.endsWith("@" + VENDOR_IDP_SUFFIX) &&
-      userType === UserType.Vendor) ||
-    (username.endsWith("@" + GOV_IDP_SUFFIX) &&
-      userType === UserType.Government)
-  ) {
-    username = username.slice(0, username.lastIndexOf("@"));
+  /**
+   * set username - stored in db as idpUsername
+   * set idpId - used as unique identifier in db, MUST be lowercase
+   */
+  let username, idpId: string;
```

Comment:
I don't think it's worth requesting a change, but technically username will have an `any` type as `idpId`'s annotation doesn't apply to it.

Reply:
you're right, single line annotation to combine types doesn't work. thanks! Change made in https://github.com/bcgov/digital_marketplace/pull/338/commits/7e35ffefbffe4ac2f9111e09bff8827ebd3cc3f0